client/my-sites/stats/stats-views/months.jsx

```diff
@@ -34,6 +34,7 @@ class Month extends PureComponent {
 
 		if ( ! isHeader && href ) {
 			page( href );
+			window.scrollTo( 0, 0 );
```

Comment:
Added a scroll to top here, as when you clicked on the month from all-time stats. The page was changing to the month's stats, but the page would be scrolled to the bottom. This can be seen in the video attached to the PR description.